libafl/src/stages/mutational.rs

```diff
@@ -92,7 +92,7 @@ where
     fn mutator_mut(&mut self) -> &mut M;
 
     /// Gets the number of iterations this mutator should run for.
-    fn iterations(&self, state: &mut Z::State) -> Result<u64, Error>;
+    fn iterations(&self, state: &mut Z::State) -> Result<usize, Error>;
```

Comment:
This changes the API a little, @tokatoka you know this code the best, anything against it?

Reply:
it's good